src/test/java/UtilsTest.java

```diff
@@ -0,0 +1,31 @@
+import junit.framework.TestCase;
+
+import junit.framework.*;
+import org.junit.Test;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import static org.junit.Assert.assertEquals;
+public class UtilsTest extends TestCase {
+
+   @Test
+   public void test() {	
+	  Utils test = new Utils(); 
```

Comment:
Since you made those functions static, you actually shouldn't need this line!

Reply:
Compiler complains when this line is omitted.